sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -519,13 +520,13 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
     for (index <- 0 until line.length) {
       if (line.charAt(index) == '\'' && !insideComment) {
         // take a look to see if it is escaped
```

Comment:
@adrian-wang Should we update the comment to reflect the newly added condition?

Reply:
Yep, maybe we can rephrase [this comment](https://github.com/apache/spark/pull/28393/files#diff-f7aac41bf732c1ba1edbac436d331a55R510) here.